.github/pull_request_template.md

```diff
@@ -0,0 +1,6 @@
+# Pull Request Checklist
+
+- [ ] Describe what changes are being made
+- [ ] Explain why and how the changes were necessary and implemented respectively
+- [ ] Reference issue with `#<ISSUE_NO>` if applicable
+- [ ] Add yourself to `CONTRIBUTORS.md` if you aren't on it already
```

Comment:
Nice. Just add this item. :)
```suggestion
- [ ] If you have changed any SCSS code, run `make release` to regenerate all CSS files.
- [ ] Add yourself to `CONTRIBUTORS.md` if you aren't on it already
```

Reply:
Thanks for that! That'll probably save me half an hour trying to figure out how to regenerate the `.content` and `.json` in resources directory.
I've re-pushed the commit incorporating the suggested change.